taxonomies/ingredients.txt

```diff
@@ -17091,7 +17095,7 @@ cz:palmu olejovou
 da:Palmeolie
 de:Palmöl, Ölpalme
 eo:Palmoleo
-es:aceite de palma, grasa de palma, aceite vegetal de palma
+es:aceite de palma, grasa de palma, aceite vegetal de palma,aceite de nuez de palma
```

Comment:
aceite de nuez de palma should be in this entry instead:

<en:palm kernel oil and fat
en:palm kernel oil, palm kernel
bs:Ulje palminog jezgra
ca:Oli de palmist
cs:Palmojádrový olej
da:Palmeolie
de:Palmkernöl, Palmkern
es:palmiste